apps/cow-tools/src/app/milkman/page.tsx

```diff
@@ -1,17 +1,5 @@
-import dynamic from "next/dynamic";
+import { redirect } from "next/navigation";
 
-import { OrdersTable } from "./components/OrdersTable";
-
-const AddressComponentWithNoSSR = dynamic(
-  () => import("./components/WalletAddress"),
-  { ssr: false },
-);
-
-export default async function Page() {
-  return (
-    <div className="flex flex-col w-full">
-      <AddressComponentWithNoSSR />
-      <OrdersTable />
-    </div>
-  );
+export default function Page() {
+  redirect("/milkman/goerli");
```

Comment:
👍 
Can you create a task to change this at the end of the project?

Reply:
Then link the issue here with a comment

Reply:
here: https://linear.app/bleu-llc/issue/BLEU-366/change-default-home-page